drone/containers/0.5/drone-config/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#FROM nrvale0/drone-debug:0.5
+FROM alpine:3.4
+
+LABEL vendor="Rancher Labs, Inc." \
+	com.rancher.version="0.5" \
+	com.rancher.repo="https://github.com/rancher/catalog-dockerfiles"
```

Comment:
@cloudnautique : Could do that but it would be redundant as I already ship the actual Dockerfile as part of the container image:

https://github.com/rancher/catalog-dockerfiles/pull/81/files#diff-89846aca6949a432cc5dfe2253c69aceR22